scalereg/settings.py

```diff
@@ -84,18 +77,21 @@
 
 ROOT_URLCONF = 'scalereg.urls'
 
-TEMPLATE_DIRS = (
-    # Put strings here, like '/home/html/django_templates' or
-    # 'C:/www/django/templates'.
-    # Always use forward slashes, even on Windows.
-    # Don't forget to use absolute paths, not relative paths.
-    '/CHANGE_THIS!!!!!__path_to/scalereg/scale_templates',
-)
-
-TEMPLATE_CONTEXT_PROCESSORS = (
-    'django.contrib.messages.context_processors.messages',
-    'django.contrib.auth.context_processors.auth',
-)
+TEMPLATES = [
+  {
+    'BACKEND': 'django.template.backends.django.DjangoTemplates',
+    'DIRS': [
+      '/var/www/django/scalereg/shellconf_templates',
```

Comment:
Let's keep the entries here completely bogus like before, so it is obvious that one need to change it.

Reply:
Missed this one. Will clean it up.